optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -84,11 +84,85 @@
       <artifactId>h2</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.btm</groupId>
+      <artifactId>btm</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
+  <properties>
+    <maven.btm.maxPoolSize>16</maven.btm.maxPoolSize>
+    <maven.hibernate.dialect>org.hibernate.dialect.H2Dialect</maven.hibernate.dialect>
+    <maven.datasource.classname>bitronix.tm.resource.jdbc.lrc.LrcXADataSource</maven.datasource.classname>
+    <maven.jdbc.driver.class>org.h2.Driver</maven.jdbc.driver.class>
+    <maven.jdbc.db.name/>
+    <maven.jdbc.db.port/>
+    <maven.jdbc.db.server/>
+    <maven.jdbc.driver.jar/>
+    <maven.jdbc.username></maven.jdbc.username>
+    <maven.jdbc.password></maven.jdbc.password>
+    <maven.jdbc.url>jdbc:h2:</maven.jdbc.url>
+    <maven.jdbc.schema>public</maven.jdbc.schema>
+  </properties>
+
+  <profiles>
+    <profile>
+      <id>database</id>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>org.jbpm</groupId>
+            <artifactId>downloaded-jdbc-driver-jar</artifactId>
+            <version>1.0</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+
+      <dependencies>
+        <!--  Add the downloaded jdbc driver jar for the database to the classpath.
+              !: <groupId>, <artifactId> and <version> are just placeholders and DO NOT MATTER! -->
+        <dependency>
+          <groupId>org.jbpm</groupId>
+          <artifactId>downloaded-jdbc-driver-jar</artifactId>
+          <scope>system</scope>
+          <systemPath>${maven.jdbc.driver.jar}</systemPath>
+        </dependency>
+      </dependencies>
```

Comment:
Just a thought: Surefire can be directly configured to use additional classpath elements. See http://maven.apache.org/surefire/maven-surefire-plugin/examples/configuring-classpath.html. This may be a better alternative to `<scope>system</scope>` dependency (which will likely be removed in Maven 4).